jira/jira_connector.bal

```diff
@@ -24,11 +24,12 @@ import ballerina/log;
 import ballerina/io;
 
 //Creates package-global Http client endpoint for jira REST API
-endpoint http:ClientEndpoint jiraHttpClientEP {targets:[{uri:WSO2_STAGING_JIRA_REST_API_ENDPOINT}],
-    chunking:http:Chunking.NEVER};
-http:HttpConnectorError connectionError;
+endpoint http:ClientEndpoint jiraHttpClientEP {
```

Comment:
Please don't create package Global endpoint.  Endpoint will be created by the end user at the point of using the endpoint.